share/fi.po

```diff
@@ -0,0 +1,298 @@
+"viesti, suoritus lakkaa. Sen on oltava yksi näistä: CRITICAL (kriittinen), "
+"ERROR (virhe), WARNING (varoitus), NOTICE (huomio), INFO (info) tai DEBUG "
+"(virheenkorjaus)."
+
+msgid "Name of profile file to load. (DEFAULT)"
+msgstr "Ladattavan profiilitiedoston nimi. (DEFAULT (oletus))"
+
+msgid "Name of configuration file to load. (TERMINATED)"
+msgstr "Ladattavan konfiguraatiotiedoston nimi. (TERMINATED (lopetettu))"
+
+msgid "Name of policy file to load. (TERMINATED)"
+msgstr "Ladattavan politiikkatiedoston nimi. (TERMINATED (lopetettu))"
+
+msgid "Strings with DS data on the form \"keytag,algorithm,type,digest\""
+msgstr ""
+"Merkkijonot, jotka sisältävät DS-tietoja muodossa \"keytag,algorithm,type,"
+"digest” (avaintunniste, algoritmi, tyyppi, tiiviste)"
+
+msgid "Print a count of the number of messages at each level"
+msgstr "Tulosta viestien lukumäärä kullakin tasolla"
+
+msgid ""
+"Boolean flag for activity indicator. Defaults to on if STDOUT is a tty, off "
+"if it is not. Disable with --noprogress."
+msgstr ""
+"Looginen lippu toiminnan merkitsemiseen. Oletusarvoisesti päällä, jos STDOUT "
+"on tty, muussa tapauksessa poissa päältä. Poistetaan käytöstä parametrilla --"
+"noprogress."
+
+msgid "Name of the character encoding used for command line arguments"
+msgstr "Komentorivin parametreissä käytetty merkistökoodaus"
+
+msgid ""
+"At the end of a run, print a summary of the times the zone's name servers "
+"took to answer."
+msgstr ""
+"Ajon lopussa tulosta yhteenveto vyöhykkeen nimipalvelinten tarvitsemista "
+"vastausajoista."
+
+msgid "Print the effective profile used in JSON format, then exit."
+msgstr "Tulosta käytössä oleva profiili JSON-muodossa ja poistu."
+
+msgid ""
+"Print the effective configuration used in JSON format, then exit. "
+"(TERMINATED)"
+msgstr ""
+"Tulosta käytössä oleva konfiguraatio JSON-muodossa ja poistu. (TERMINATED "
+"(lopetettu))"
+
+msgid "Print the effective policy used in JSON format, then exit. (TERMINATED)"
+msgstr ""
+"Tulosta käytössä oleva politiikka JSON-muodossa ja poistu. (TERMINATED "
+"(lopetettu))"
+
+msgid ""
+"Local IP address that the test engine should try to send its requests from."
+msgstr ""
+"Paikallinen IP-osoite, jota testimoottorin tulisi yrittää käyttää "
+"pyyntöjensä lähettämiseen."
+
+#, perl-format
+msgid ""
+"Warning: setting locale category LC_MESSAGES to %s failed (is it installed "
+"on this system?).\n"
+"\n"
+msgstr ""
+"Varoitus: arvoa %s ei voitu asettaa lokaalikategorian LC_MESSAGES arvoksi "
+"(onko sitä asennettu tähän järjestelmään?).\n"
+"\n"
+
+#, perl-format
+msgid ""
+"Warning: setting locale category LC_CTYPE to %s failed (is it installed on "
+"this system?).\n"
+"\n"
+msgstr ""
+"Varoitus: arvoa %s ei voitu asettaa lokaalikategorian LC_CTYPE arvoksi (onko "
+"sitä asennettu tähän järjestelmään?).\n"
+"\n"
+
+#, perl-brace-format
+msgid "Address {address} cannot be used as source address for DNS queries.\n"
+msgstr ""
```

Comment:
Untranslated. Needs to be fixed by translator.

Reply:
"Osoitetta {address} ei voi käyttää lähdeosoitteena DNS kyselyille.\n"

Reply:
Thanks. Included.